cmd/scheduled-feed/main.go

```diff
@@ -97,8 +98,25 @@ func main() {
 		pub:       pub,
 		delta:     delta,
 	}
+
+	if appConfig.Timer {
+		cronjob := cron.New()
+		crontab := fmt.Sprintf("@every %s", delta.String())
+		log.Printf("Running a timer %s", crontab)
+		cronjob.AddFunc(crontab, func() { cronrequest(appConfig.HttpPort) })
+		cronjob.Start()
+	}
+
 	http.Handle("/", handler)
 	if err := http.ListenAndServe(fmt.Sprintf(":%v", appConfig.HttpPort), nil); err != nil {
 		log.Fatal(err)
 	}
 }
+
+func cronrequest(port int) {
+	resp, err := http.Get(fmt.Sprintf("http://localhost:%v", port))
+	if err != nil {
+		log.Printf("http request failed")
```

Comment:
I would add the error message to the log

Reply:
This raises another question, should this be a `log.Fatal()`, as it stands if a scheduled call was to fail (or be delayed waiting on a timeout) then upto 5 minutes of events could be naively missed per cron.

Reply:
`err` added to the log call, thanks